openfifth/aspen-discovery
# Summon - increase load speed

This patch adds the ability to use only book covers generated by Aspen or to include those available from the API when carrying out a Summon Search.  Choosing to use generated covers only increases the speed at which Summon loads results. 

TEST PLAN:
(After applying patch).
1. Carry out a Summon Search as usual. You will notice that you are returned book covers that are generated by Aspen.
2. Navigate to Primary Configurations -> Library Systems and then edit whichever Library is your default.
3. Scroll to the Summon Section and check the tick box that is labelled, 'Show Available Covers in Summon.'
4. Click to save your changes and repeat your search.
5. Notice that the covers you are returned are now a mix of generated covers and actual book covers from the API - note that your results may load more slowly with this option enabled.
